website/docs/r/transcribe_vocabulary.html.markdown

```diff
@@ -0,0 +1,81 @@
+---
+subcategory: "Transcribe"
+layout: "aws"
+page_title: "AWS: aws_transcribe_vocabulary"
+description: |-
+  Terraform resource for managing an AWS Transcribe Vocabulary.
+---
+
+# Resource: aws_transcribe_vocabulary
+
+Terraform resource for managing an AWS Transcribe Vocabulary.
+
+## Example Usage
+
+### Basic Usage
+
+```terraform
+resource "aws_s3_bucket" "example" {
+  bucket        = "example-vocab-123"
+  force_destroy = true
+}
+
+resource "aws_s3_object" "object" {
+  bucket = aws_s3_bucket.example.id
+  key    = "transcribe/test1.txt"
+  source = "test.txt"
+}
+
+resource "aws_transcribe_vocabulary" "example" {
+  vocabulary_name     = "example"
+  language_code       = "en-US"
+  vocabulary_file_uri = "s3://${aws_s3_bucket.example.id}/${aws_s3_object.object.key}"
+
+  tags = {
+    tag1 = "value1"
+    tag2 = "value3"
+  }
+
+  depends_on = [
+    aws_s3_object.object
+  ]
+}
+```
+
+## Argument Reference
+
+The following arguments are required:
+
+* `language_code` - (Required) The language code you selected for your vocabulary.
+* `vocabulary_file_uri` - (Required) The Amazon S3 location (URI) of the text file that contains your custom vocabulary.
+* `vocabulary_name` - (Required) The name of the Medical Vocabulary.
+
+The following arguments are optional:
+
+* `phrases` - (Optional) - A list of terms to include in the vocabulary. Conflicts with `vocabular_file_uri`
+* `vocabulary_file_uri` - (Optional) The Amazon S3 location (URI) of the text file that contains your custom vocabulary. Conflicts wth `phrases`.
+* `tags` - (Optional) A map of tags to assign to the MedicalVocabulary. If configured with a provider [`default_tags` configuration block](/docs/providers/aws/index.html#default_tags-configuration-block) present, tags with matching keys will overwrite those defined at the provider-level.
+
+## Attributes Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - Name of the MedicalVocabulary.
+* `arn` - ARN of the MedicalVocabulary.
+* `download_uri` - Generated download URI.
+
+## Timeouts
+
+`aws_transcribe_medicalvocabulary` provides the following [Timeouts](https://www.terraform.io/docs/configuration/blocks/resources/syntax.html#operation-timeouts) configuration options:
+
+* `create` - (Optional, Default: `30m`)
+* `update` - (Optional, Default: `30m`)
+* `delete` - (Optional, Default: `30m`)
+
+## Import
+
+Transcribe Vocabulary can be imported using the `vocabulary_name`, e.g.,
+
+```
+$ terraform import aws_transcribe_vocabulary.example rft-8012925589
```

Comment:
`rft-8012925589` looks more like an ID than a name to me. If `vocabulary_name` is the import ID I suggest a more human-friendly example.